plugin/src/ChangeBatcher/encodePatchCreate.lua

```diff
@@ -15,28 +15,30 @@ local  function createInstanceSnapshot(instanceMap,instance)
 		table.insert(children,createInstanceSnapshot(instanceMap,child))
 	end
 
-	local properties = RbxDom.findAllNoneDefaultPropertiesEncoded(instance)
-	properties.Name = nil
-	properties.ClassName = nil
-
-	local attributes = nil
-	if #properties.Attributes ~= 0 then
-		attributes = properties.Attributes
-	end
-
-	properties.Attributes = nil
-	if #properties == 0 then
-		properties = nil
+	local success, properties = RbxDom.findAllNoneDefaultPropertiesEncoded(instance)
+	if success
```

Comment:
`if success then`? Did you test this?

Reply:
everyting exept that final change thanks for checking